GRPC install instructions and production warning (#74)
awakesecurity/gRPC-haskell: README.md

-----------

This library is a fork of https://github.com/aloiscochard/grpc-haskell that we

> have extended and released under the same [`LICENSE`](./LICENSE). It's not ready for production use and use it with caution.

Reviewer: We do use this project quite a bit in production, so I do think it is production ready. However, we have not polished the project, it lacks good documentation, and some dependencies have fallen far behind so I think it is accurate to say:

```suggestion
have extended and released under the same [`LICENSE`](./LICENSE). Please use with caution in production.
```

  sha256 "efad782944da13d362aab9b81f001b7b8b1458794751de818e9848c47acd4b32"
  ```

> Installing GRPC on Unix

Reviewer: Instead of recommending that the user clone the repo and update the submodules, I think we should instead recommend downloading the release package from the project's release page: https://github.com/grpc/grpc/releases.

e.g.

```
1. Download the 1.2 release: `curl -O https://github.com/grpc/grpc/archive/v1.2.0.tar.gz`
...
```

Even better is if we can suggest installation via a specific distro's package manager but I don't know how many versions of grpc core are available for Debian, etc.
